src/truefoundry_sdk/core/parse_error.py

```
@@ -0,0 +1,36 @@
+
+    headers: Optional[Dict[str, str]]
+    status_code: Optional[int]
+    body: Any
+    cause: Optional[Exception]
+
+    def __init__(
+        self,
+        *,
+        headers: Optional[Dict[str, str]] = None,
+        status_code: Optional[int] = None,
+        body: Any = None,
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `body`

(ANN401)